src/contacts/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
+# This file is autogenerated by pip-compile with python 3.8
```

Comment:
Thought: I'm going to guess this `3.8` vs `3.10` difference is fine — since it's not a major version difference.